src/skodun/cli.py

```
@@ -977,9 +977,13 @@ def _cmd_stats(args) -> int:
     except BaseException as e:
         return _emit(f"skodun stats: could not open the store: {e!r}", 2)
     with store:
+        try:
+            repo = _repo_root(Path.cwd())
+        except gitio.GitError:
```

Ruff (0.16.3)

[error] Undefined name `gitio`

(F821)